orders/tests.py

```diff
@@ -0,0 +1,99 @@
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now()
+TOTAL_PRICE = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def order():
+    return Order(dog_owner_id=DogOwner.create(email='test_dog_owner556@gmail.com',
```

Comment:
why not using DogOwner and DayCare that were created in the data migrations? in the current situation you might have conflicts with the data that was created there

Reply:
> why not using DogOwner and DayCare that were created in the data migrations? in the current situation you might have conflicts with the data that was created there

Good idea. I'll change it.